gradle.properties

```diff
@@ -1,5 +1,5 @@
 version=0.6.0-SNAPSHOT
```

Comment:
`0.6.0-Beta-SNAPSHOT`.

Also, note how we are using `Beta` instead of `BETA` all over the Kotlin, let's stick to the single naming scheme